samples/client/petstore/groovy/build.gradle

```diff
@@ -19,8 +18,6 @@ buildscript {
 repositories {
     mavenCentral()
     mavenLocal()
-    mavenCentral(artifactUrls: ['https://maven.springframework.org/milestone'])
```

Comment:
You might need this, I don't know.

Reply:
The project compiles without issue. If anyone needs this for whatever reasons, we can certainly add it back.